volttrontesting/platform/test_instance_setup.py

```diff
@@ -8,11 +8,17 @@
 from volttron.platform.instance_setup import _is_agent_installed
 from volttron.utils import get_hostname
 from volttron.platform.agent.utils import is_volttron_running
-from volttrontesting.fixtures.rmq_test_setup import create_rmq_volttron_setup
-from volttrontesting.utils.platformwrapper import create_volttron_home
+from volttron.platform import is_rabbitmq_available
 
-HAS_RMQ = is_rabbitmq_available()
 RMQ_TIMEOUT = 600
+HAS_RMQ = is_rabbitmq_available()
+if HAS_RMQ:
+    from volttrontesting.fixtures.rmq_test_setup import create_rmq_volttron_setup
+else:
```

Comment:
The if part makes sense but in the else why should the entire module be skipped. The module has both rmq and zmq tests.